theme/css/zend-expressive.css

```diff
@@ -0,0 +1,48 @@
+/* Headers changes */
+body {
+     font-family: 'Open Sans';
```

Comment:
I may change this back. I specifically chose a serif font for purposes of readability. (Full disclosure: I used to work in the print industry, doing book, magazine, and catalog layout, so I have some experience with fonts!)